python/ray/train/trainer.py

```diff
@@ -135,6 +136,16 @@ def __init__(
         logdir: Optional[str] = None,
         max_retries: int = 3,
     ):
+        warnings.warn(
+            "The `ray.train.Trainer` API will be deprecated in Ray "
+            "2.0, and will be replaced by Ray AI Runtime (Ray AIR). Ray AIR ("
+            "https://docs.ray.io/en/master/ray-air/getting-started.html) will "
```

Comment:
Should we point users to latest versions of the docs (_i.e_, https://docs.ray.io/en/latest) as opposed to master?

Reply:
Updated!